package.json

```
@@ -37,7 +37,7 @@
     "husky": "^3.0.4",
     "npm-run-all": "^4.1.3",
     "patch-package": "^6.1.2",
-    "prettier": "^1.18.1",
+    "prettier": "^2.8.2",
```

Patch releases of Prettier introduce formatting changes, so it’s recommended to pin an exact version.

```suggestion
    "prettier": "2.8.2",
```

---

Good call-out: https://github.com/actualbudget/actual/pull/442/commits/afdfd3c6b188e160242ba5f8839d177ece53e7f1